lib/virt_autotest/utils.pm

```diff
@@ -379,7 +379,7 @@ sub ensure_online {
         }
         unless ($skip_ssh == 1) {
             # Wait for ssh to come up
-            die "$guest does not start ssh" if (script_retry("nmap $guest -PN -p ssh | grep open", delay => 15, retry => 12) != 0);
+            die "$guest does not start ssh" if (script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 12, timeout => 360) != 0);
```

Comment:
It seems that you skip ping for 15sp5 pv guest, and also extend timeout for detecting ssh port ?

Reply:
yes, during calling `ensure_online()` for 15-SP5 PV guest system, just only skip ping and also also extend timeout for detecting ssh port. Actually,  skip ping just only save execute time during calling `ensure_online()` for each 15-SP5 PV guest system. meanwhile, there is enough to extend timeout for detecting ssh port for 15-SP5 PV guest system due to bsc#1206250. imho.